target/linux/mvebu/cortexa9-neon/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# Copyright (C) 2014-2016 OpenWrt.org
+# Copyright (C) 2016 LEDE-Project.org
+#
```

Comment:
I'd personally remove those generic copyright entries here and for the files below (just an optional remark, as I'm aware you just copied)